tests/performance/src/main/java/org/mule/FlowBenchmark.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule;
+
+import static java.lang.Class.forName;
+import static java.util.Collections.singletonList;
+import static java.util.concurrent.TimeUnit.MICROSECONDS;
+import static org.mule.runtime.core.api.lifecycle.LifecycleUtils.stopIfNeeded;
+import org.mule.runtime.api.exception.MuleException;
+import org.mule.runtime.core.DefaultEventContext;
+import org.mule.runtime.core.api.Event;
+import org.mule.runtime.core.api.MuleContext;
+import org.mule.runtime.core.api.message.InternalMessage;
+import org.mule.runtime.core.api.processor.strategy.ProcessingStrategyFactory;
+import org.mule.runtime.core.api.scheduler.SchedulerService;
+import org.mule.runtime.core.construct.Flow;
+import org.mule.tck.TriggerableMessageSource;
+
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.OutputTimeUnit;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.TearDown;
+
+
+@OutputTimeUnit(MICROSECONDS)
+public class FlowBenchmark extends AbstractBenchmark {
+
+  public static final String TEST_PAYLOAD = "test";
+  public static final String TEST_CONNECTOR = "test";
+
+  private MuleContext muleContext;
+  private Flow flow;
+  private TriggerableMessageSource source;
+
+  @Param({"org.mule.runtime.core.processor.strategy.LegacySynchronousProcessingStrategyFactory",
```

Comment:
i think i already asked this, but can't you put the class instead of the string?